x/config/config.go

```diff
@@ -0,0 +1,117 @@
+package config
```

Comment:
This is pretty new so I don't know if we should commit to absolutely no breaking changes after the first release. We wouldn't want to cut a YARPC 2.0 because we realized that something in this API didn't quite work right after test driving it.